itou/siae_evaluations/templatetags/siae_evaluations_siae_tags.py

```diff
@@ -85,3 +89,78 @@ def evaluated_job_application_state_for_siae(evaluated_job_application):
         if real_state_priority < submitted_state_priority:
             state = submitted_state
     return get_employer_badges(not evaluated_siae.reviewed_at)[state]
+
+
+def get_labor_inspector_badges(amicable_stage, submission_freezed):
+    NOT_SUBMITTED = danger_badge("Justificatifs non transmis")
+    submission_freezed_badges = {
+        EvaluatedJobApplicationsState.PENDING: NOT_SUBMITTED,
+        EvaluatedJobApplicationsState.PROCESSING: NOT_SUBMITTED,
+        EvaluatedJobApplicationsState.UPLOADED: NOT_SUBMITTED,
+        EvaluatedJobApplicationsState.SUBMITTED: TODO_BADGE,
```

Comment:
que donnerait `EvaluatedJobApplicationsState.SUBMITTED: PENDING_AFTER_REVIEW_BADGE if submission_freezed and amicable_stage else TODO_BADGE` à la place des `if` imbriqués ?

Reply:
C’est ce que je ne veux pas, je veux justement que les `if` de haut niveau définissent l’intégralité des états possibles, sans avoir à repasser derrière pour surcharger. L’idée est de coller aux phases:
- amiable
- amiable, soumissions bloquées
- contradictoire
- contradictoire, soumissions bloquées
- finale

Le raisonnement est ainsi « quand on est en phase amiable, comment est affiché le statut X à l’utilisateur ». Plutôt que la version actuelle qui part de l’état de la candidature, et change en fonction de la phase, de manière pas toujours évidente.